Lesson 5

alekseynarizhniy opened the pull request:
- fixed names and a little optimized code
- added two custom directives according to homework
- changed type any

alekseynarizhniy committed:
- fixed names and a little optimized code
- added two custom directives according to homework

alekseynarizhniy committed:
- changed type any

iYanovoy commented on src/app/components/component.module.ts:
```diff
@@ -5,17 +5,25 @@ import { MatPaginatorModule } from '@angular/material/paginator';
 
 import { reducer } from '../reducers/bucket.reducer';
 
+import { HightlightShadowDirective } from "../directives/hightlightshadow.directive";
```
Be consistent, use single quotes as at previous lines

iYanovoy commented on src/app/components/dialog-product/dialog-product.component.scss:
```diff
@@ -0,0 +1,47 @@
+  position: relative;
+  color: white;
+
+  table {
```
Do not use element as CSS selector

iYanovoy commented on src/app/directives/dialog.directive.ts:
```diff
@@ -0,0 +1,24 @@
+  selector: '[appDialog]',
+})
+export class DialogDirective {
+  @Input() appDialog!: ProductWrapper;
```
Name of Input doesn't response target (it works with product/dialog data)

alekseynarizhniy committed:
- renamed pipe to camelCase
- changed imports to one style
- renamed dialog product directive
- added clases for table and elements inside
